prerequisites.sh

```diff
@@ -22,7 +22,7 @@ if [ -d $LIBS_DIR ] && [ "$(ls -A $LIBS_DIR)" ]; then
   INSTALLED=`tr -d '\n' < $LIBS_DIR/.VERSION`
   echo -e "The installed version of libsemigroups is v$INSTALLED"
   LEAST=`echo -e "$VERS\n$INSTALLED" | sort -V | head -n1`
```

Comment:
I think only this one needs to be `echo -e`, to convert the `\n` into a newline character. The rest could all be without the `-e`.